doc/guides/writing-and-running-benchmarks.md

```diff
@@ -202,12 +202,11 @@ For analysing the benchmark results use the `compare.R` tool.
 ```console
 $ cat compare-pr-5134.csv | Rscript benchmark/compare.R
 
-                                                                                      improvement confidence      p.value
-string_decoder/string-decoder.js n=250000 chunk=1024 inlen=1024 encoding=ascii           12.46 %         *** 1.165345e-04
-string_decoder/string-decoder.js n=250000 chunk=1024 inlen=1024 encoding=base64-ascii    24.70 %         *** 1.820615e-15
-string_decoder/string-decoder.js n=250000 chunk=1024 inlen=1024 encoding=base64-utf8     23.60 %         *** 2.105625e-12
-string_decoder/string-decoder.js n=250000 chunk=1024 inlen=1024 encoding=utf8            14.04 %         *** 1.291105e-07
-string_decoder/string-decoder.js n=250000 chunk=1024 inlen=128  encoding=ascii            6.70 %           * 2.928003e-02
+                                                                                             confidence improvement accuracy (*)    (**)   (***)
```

Comment:
nit: The last three columns are not aligned in [private branch](https://github.com/xsbchen/node/blob/master/doc/guides/writing-and-running-benchmarks.md#comparing-nodejs-versions):
![screen shot 2018-04-28 at 2 05 28 pm](https://user-images.githubusercontent.com/16024985/39400874-9bc1ff3a-4aed-11e8-9ab1-4fb959cc7d56.png)

Reply:
Copy from console log，should I fixed it in doc?

Reply:
doc might be using different whitespaces as compared to `console.log`
It's a nit, check if you can add some spaces to align the columns in the doc. When I ran benchmark script, the last three columns were aligned.

Reply:
done

Reply:
Thanks! 😄